wpimath/src/main/java/edu/wpi/first/math/controller/ArmFeedforward.java

```diff
@@ -69,6 +69,42 @@ public ArmFeedforward(double ks, double kg, double kv) {
     this(ks, kg, kv, 0);
   }
 
+  /**
+   * Returns the static gain.
+   *
+   * @return The static gain, in volts.
+   */
+  public double getKs() {
+    return ks;
+  }
+
+  /**
+   * Returns the gravity gain.
+   *
+   * @return The gravity gain, in volts.
+   */
+  public double getKg() {
+    return kg;
+  }
+
+  /**
+   * Returns the velocity gain.
+   *
+   * @return The velocity gain, in volt seconds per radian.
+   */
+  public double getKv() {
+    return kv;
+  }
+
+  /**
+   * Returns the acceleration gain.
+   *
+   * @return The acceleration gain, in volts seconds² per radian.
```

Comment:
It's a lot clearer to write the units as V/(rad/s²), because it's volts per angular acceleration.